source/manual/howto-add-organisation-brand-colour.html.md

```diff
@@ -7,42 +7,53 @@ parent: "/manual.html"
 related_repos: [whitehall, government-frontend]
 ---
 
-An organistion's brand colour is used to style their organisation page (such as
-dividing lines, icons and link text). For example
-see the [DCMS organisation page](https://www.gov.uk/government/organisations/department-for-digital-culture-media-sport).
+An organisation's brand colour is used to style their organisation page (such as dividing lines, icons and link text). For example see the [DCMS organisation page](https://www.gov.uk/government/organisations/department-for-digital-culture-media-sport).
 
-The colour is set as an option under a drop down field called "brand colour" when
-creating or editing an [organisation page in Whitehall Publisher](https://whitehall-admin.publishing.service.gov.uk/government/admin/organisations).
+The colour is set as an option under a drop down field called "brand colour" when creating or editing an [organisation page in Whitehall Publisher](https://whitehall-admin.publishing.service.gov.uk/government/admin/organisations).
 
-## 1. Add the brand colour in GOV.UK Frontend
+[Documentation in govuk_publishing_components](https://github.com/alphagov/govuk_publishing_components/blob/main/docs/component_branding.md) explains how brand colours work in the code, for reference.
+
+## 1. Check the colour passes WCAG contrast requirements
+
+If an organisation's brand colour changes or a new one is required, check that it is of a high enough contrast against a white background to be accessible. Use a tool such as the [WebAIM Contrast Checker](https://webaim.org/resources/contrastchecker/) to determine this.
```

Comment:
Should this include what WCAG level we target - e.g. should the colour meet WCAG AA or AAA?